docs/content/direct/user-guide.md

```diff
@@ -34,7 +34,13 @@ The following are applied to every workload object.
 
 ## Built-in transformations of specific kinds of workload object
 
-In a `Service` (core API group) object, remove the following fields from `spec`: `clusterIP`, `clusterIPs`, `ipFamilies`, `externalTrafficPolicy`, `internalTrafficPolicy`, `ipFamilyPolicy`, `sessionAffinity`. Also remove the `nodePort` field from every port unless the annotation `kubestellar.io/annotations/preserve=nodeport` is present.
+In a `Service` (core API group) object:
+
+1. remove the following fields from `spec`: `ipFamilies`, `externalTrafficPolicy`, `internalTrafficPolicy`, `ipFamilyPolicy`, `sessionAffinity`. Also remove the `nodePort` field from every port unless the annotation `kubestellar.io/annotations/preserve=nodeport` is present;
```

Comment:
Is it clear we mean that KS automatically does that ?  "remove"  might  be interpreted as we asked the user to remove these fields (I know it's under "Built-in transformations ")  - but maybe we should mention it explicitly ?

Reply:
As long as you think it's clear